index.html

```diff
@@ -140,7 +142,10 @@ <h1 class="pull-left"><span class="octicon octicon-logo-github" aria-label="GitH
                 <strong>Warning!</strong> Live Preview has been disabled due to low API rate limit remaining.
               </div>
               <div class="form-group">
-                <textarea id="code" class="code form-control" rows="2" readonly></textarea>
+                <textarea id="code" class="code form-control" rows="3" readonly></textarea>
+                <button class="btn" data-clipboard-target="#code">
```

Comment:
*Did you test your code at all?*

`clipboard.js` does not bind any event to any class by default. It has to be manually instantiated.